### DISCUSSION TITLE
第5节课笔记(4班)

### DISCUSSION THREAD

**Discussion by FlamingoPg** [InternLM/Tutorial#317] [笔记]
请大家把第五节课笔记放到这里哦~

**Comment by zhjunqin** [comment]
zhjunqin: https://blog.csdn.net/zhjunqin/article/details/135587979?spm=1001.2014.3001.5502

**Comment by Amo-Zeng** [comment]
用了算力，还是交一下作业：
第五次课程只到了生成api,算力用尽
![350](https://github.com/InternLM/tutorial/assets/51903518/37d4ec49-f3b4-4054-a3e0-893ad168e289)
其他的也一并放在这里：
第四次xtuner：根据首页命令对7b基座模型针对alpaca数据微调了两个epoch
![349](https://github.com/InternLM/tutorial/assets/51903518/659ab315-8a39-4305-9de8-11a07e8ea82d)
第三次langchain：教程的embedding模型对中文很糟糕，我换成了text2vec并把《明朝那些事》放到数据库了
![348](https://github.com/InternLM/tutorial/assets/51903518/0556d1fa-1821-45b3-902b-86bac4449429)
![347](https://github.com/InternLM/tutorial/assets/51903518/21da8fea-d4dc-4b7c-94d4-085462c4c0f2)
多模态等
![346](https://github.com/InternLM/tutorial/assets/51903518/2a000f3a-e2f9-4162-8742-e005d08a949f)
![345](https://github.com/InternLM/tutorial/assets/51903518/6cbe125f-fb05-4a21-89ae-87f095020ed0)
![344](https://github.com/InternLM/tutorial/assets/51903518/a5623529-9277-4558-8d28-703bf6cddd59)

**Comment by onlyzer0** [comment]
onlyzer0:https://blog.csdn.net/qq_42907812/article/details/135611313?spm=1001.2014.3001.5501

**Comment by phbst** [comment]
https://thankful-archeology-738.notion.site/830069153252478aae833ab58533c47f?pvs=4
